microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode.md

```diff
@@ -0,0 +1,179 @@
+
+1. Click **Experiments** and scroll down to the **Focus Mode** checkbox.
+
+   If the checkbox is selected, Focus Mode is already enabled. Select or clear it to enable or disable Focus Mode.
+
+    ![Screenshot of the Settings page in DevTools, showing the Experiments tab, scrolled down to the Focus Mode checkbox](media/focus-mode-pref.png)
+
+1. Close the **Settings** page. If you changed the Focus Mode experiment state, click the **Reload DevTools** button for the change to take effect.
+
+
+<!-- ====================================================================== -->
+## Main differences with the default user interface
+
+Here are the main differences between the default user interface of DevTools and Focus Mode.
+
+* **Activity Bar**
+
+  The **Activity Bar** is the most visible difference with the default user interface. Initially, it appears as a vertical sidebar to the left of the DevTools window. The **Activity Bar** contains icons used to open tools and access help and settings.
+
+  It replaces the main toolbar from the default user interface.
+
+  ![Screenshot of the activity bar in Focus Mode](media/focus-mode-activity-bar.png)
+
+* **Quick View**
+
+  The **Quick View** list contains all tools, and is used to display a tool alongside the one already selected from the **Activity Bar**.
+
+  **Quick View** replaces the drawer toolbar from the default user interface.
+
+  ![Screenshot of the quick view list in Focus Mode with a tool selected and opened in the bottom pane](media/focus-mode-quick-view.png)
+
+* DevTools customization and settings
+
+  Several customization features as well as the DevTools settings are now grouped together under a single menu available from the **Customize and control DevTools** (**...**) button in the **Activity Bar**. These features and settings appear in multiple locations in the default user interface.
+
+  * **Dock location** allows to change the placement of DevTools in the browser window.
+  * **Activity Bar location** allows to change the placement of the **Activity Bar** within DevTools.
+  * **Themes** allows to change the color theme.
+  * **Settings** provides access to the DevTools settings.
+  * **Keyboard shortcuts** allows to review and change keyboard shortcuts.
+
+  ![Screenshot of the menu in Focus Mode](media/focus-mode-menu.png)
+
+* Help links
+
+  The **Help** menu contains links to the DevTools documentation and release notes and a button to send feedback to the team. These links and buttons were previously nested under **Customize and control DevTools** > **Help** in the default user interface.
+
+  ![Screenshot of the help menu in Focus Mode](media/focus-mode-help.png)
+
+
+<!-- ====================================================================== -->
+## Open tools from the Activity Bar
+
+By default, the **Activity Bar** contains the following tools:
+
+* Icon tools:
+   *  **Inspect tool** (![Inspect tool icon.](../media/inspect-tool-icon-light-theme.png)) toggle button.
```

Comment:
Yes, this button toggles a new mode which can be used regardless of which tool is selected. (Mouse over content in the web page with this button selected to see its functionality.)

In my opinion, Inspect (and Device Mode below it) should be considered "modes" rather than "tools" because they allow different content views independent of tools such as Elements and Console.

Reply:
Great, fixing